modules/localstack/src/main/java/org/testcontainers/containers/localstack/LocalStackContainer.java

```diff
@@ -28,16 +29,15 @@
  */
 public class LocalStackContainer extends GenericContainer<LocalStackContainer> {
 
-    public static final String VERSION = "0.8.6";
 
     private final List<Service> services = new ArrayList<>();
 
     public LocalStackContainer() {
-        this(VERSION);
+        this(TestcontainersConfiguration.getInstance().getLocalStackImage().split(":")[1]);
```

Comment:
by default, we should use the provided image

Reply:
This would use the default version, unless the user has already overridden the property, which is then exactly what this PR is addressing.

Reply:
If the user overrides the image, we should use it "as it" and pass to `super(TestcontainersConfiguration.getInstance().getLocalStackImage())`.

The image might not have `:` at all